src/styles/base/_css-reset.scss

```diff
@@ -9,8 +9,8 @@
  */
```

Comment:
We should make this a mixin and include it in all component css. That should do the trick.